<ui version="4.0" >
 <class>BoundaryConditionsCoalInletForm</class>
 <widget class="QWidget" name="BoundaryConditionsCoalInletForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBoxFlowTemp" >
     <property name="title" >
      <string>Flows and temperatures</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QGridLayout" name="_2" >
        <item row="0" column="0" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Inlet type : </string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3" >
         <widget class="QComboBox" name="comboBoxTypeInlet" >
          <property name="minimumSize" >
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="4" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxAir" >
        <property name="title" >
         <string>Air</string>
        </property>
        <layout class="QGridLayout" name="_3" >
         <item row="0" column="0" >
          <layout class="QGridLayout" name="_4" >
           <item row="0" column="0" >
            <widget class="QComboBox" name="comboBoxAirVelocity" >
             <property name="minimumSize" >
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLineEdit" name="lineEditAirVelocity" >
             <property name="maximumSize" >
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2" >
            <widget class="QLabel" name="labelUnitVelocityAir" >
             <property name="text" >
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="4" >
            <widget class="QLabel" name="labelTempAir" >
             <property name="text" >
              <string>Temperature</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5" >
            <widget class="QLineEdit" name="lineEditTemperature" >
             <property name="maximumSize" >
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="6" >
            <widget class="QLabel" name="labelUnitTemp" >
             <property name="text" >
              <string>K</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxCoal" >
        <property name="title" >
         <string>Coal</string>
        </property>
        <layout class="QVBoxLayout" name="_5" >
         <item>
          <widget class="QTableView" name="tableViewCoal" >
           <property name="alternatingRowColors" >
            <bool>true</bool>
           </property>
           <property name="selectionBehavior" >
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxDirection" >
        <property name="title" >
         <string>Direction</string>
        </property>
        <layout class="QGridLayout" name="_6" >
         <item row="0" column="0" >
          <layout class="QGridLayout" name="_7" >
           <item row="0" column="0" >
            <layout class="QGridLayout" name="_8" >
             <item row="0" column="0" >
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="1" >
              <widget class="QComboBox" name="comboBoxDirectionVelocityCoal" >
               <property name="minimumSize" >
                <size>
                 <width>200</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="2" >
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="1" column="0" >
            <widget class="QFrame" name="frameCoalVelocity" >
             <property name="minimumSize" >
              <size>
               <width>16</width>
               <height>42</height>
              </size>
             </property>
             <property name="frameShape" >
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Raised</enum>
             </property>
             <property name="lineWidth" >
              <number>0</number>
             </property>
             <layout class="QGridLayout" name="_9" >
              <item row="0" column="0" >
               <layout class="QGridLayout" name="_10" >
                <item row="0" column="0" >
                 <widget class="QLabel" name="labelX" >
                  <property name="text" >
                   <string>X</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" >
                 <widget class="QLineEdit" name="lineEditXVelocityCoal" >
                  <property name="maximumSize" >
                   <size>
                    <width>80</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3" >
                 <widget class="QLabel" name="labelY" >
                  <property name="text" >
                   <string>Y</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="4" >
                 <widget class="QLineEdit" name="lineEditYVelocityCoal" >
                  <property name="maximumSize" >
                   <size>
                    <width>80</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="0" column="5" >
                 <widget class="QLabel" name="labelUnitY" >
                  <property name="text" >
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="0" column="6" >
                 <widget class="QLabel" name="labelZ" >
                  <property name="text" >
                   <string>Z</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="7" >
                 <widget class="QLineEdit" name="lineEditZVelocityCoal" >
                  <property name="maximumSize" >
                   <size>
                    <width>80</width>
                    <height>16777215</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="0" column="8" >
                 <widget class="QLabel" name="labelSlideUnitZ" >
                  <property name="text" >
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxCoalMass" >
        <property name="title" >
         <string>Mass distribution</string>
        </property>
        <layout class="QGridLayout" name="_11" >
         <item row="0" column="0" >
          <widget class="QTableView" name="tableViewCoalMass" >
           <property name="alternatingRowColors" >
            <bool>true</bool>
           </property>
           <property name="selectionBehavior" >
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
